addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -338,7 +339,40 @@ export class WebsitePreview extends Component {
         this.title.setParts({ action: currentTitle });
     }
 
-    _onPageLoaded() {
+    _onPageLoaded(ev) {
+        // FIX Chrome-only. If you have the backend in a language A but the
+        // website in English only, you can 1) modify a record's (event,
+        // product...) name in language A (say "New Name").
+        // 2) visit the page `/new-name-11` => the server will redirect you to
+        // the English page `/origin-11`, which is the only one existing.
+        // Chrome caches the redirection.
+        // 3) give the same name in English as in language A, try to visit
+        // => the server now wants to access `/new-name-11`
+        // => Chrome uses the cache to redirect `/new-name-11` to `/origin-11`,
+        // => the server tries to redirect to `/new-name-11` => start again 🔄.
+        // If we detect that behavior, we reload the iframe with a new query
+        // parameter, making the URL brand-new for Chrome.
+        if (navigator.userAgent.toLowerCase().includes("chrome") && !this.iframeReloaded) {
+            try {
+                this.iframe.el.contentWindow.location.href;
+            } catch (err) {
+                const msgCheck = `Failed to read a named property 'href' from 'Location': Blocked a frame with origin "${location.origin}" from accessing a cross-origin frame.`;
+                if (err.name === "SecurityError" && err.message === msgCheck) {
+                    ev.stopImmediatePropagation();
+                    this.iframe.el.src = `${this.iframe.el.src}${encodeURIComponent("?iframe_reload=1")}`;
+                    this.iframeReloaded = true;
+                    this.notificationService.add(
+                        this.env._t("You should empty your browser's cache."),
+                        {
+                            type: "warning",
+                            title: this.env._t("Oops, we avoided a crash."),
+                            sticky: true,
+                        }
+                    );
```

Comment:
We should not let the user knows about this but just do it for him, a "browser cache" has no meaning for 99% of people, it's a really advanced and technical thing.

People changing the name of a record don't expect the software to show an error, it should be silent